src/save.ts

```diff
@@ -45,7 +46,9 @@ async function run() {
     } catch {}
 
     try {
-      await cleanTarget(packages);
+      for (const target of targets) {
+        await cleanTarget(target, packages);
```

Comment:
maybe move the try/catch into the for. better not to skip all following target dirs just because we hit a file not found error or similar.